src/PaywallFooterView.tsx

```diff
@@ -0,0 +1,3 @@
+// import React from 'react';
+import {requireNativeComponent} from 'react-native';
+export const PaywallFooterView = requireNativeComponent('RCPaywallFooterView');
```

Comment:
It could be interesting to make this a container view, that contains this `PaywallFooterView` and allows the developer to pass us their UI so we can handle the positioning of the view for them